packages/lesswrong/components/common/SearchBar.tsx

```diff
@@ -60,36 +62,23 @@ const styles = (theme: ThemeType): JssStyles => ({
       fontSize: 'inherit',
       "-webkit-appearance": "none",
       cursor: "text",
-      borderRadius:5,
-
-      [theme.breakpoints.down('tiny')]: {
-        backgroundColor: theme.palette.grey[200],
-        zIndex: theme.zIndexes.searchBar,
-        width:110,
-        height:36,
-        paddingLeft:10
-      },
+      borderRadius: 5,
     },
     "&.open .ais-SearchBox-input": {
       display:"inline-block",
     },
   },
   searchIcon: {
     position: 'fixed',
-    color: isEAForum ? theme.palette.grey[600] : undefined,
-    margin: '12px',
-    '&:hover': {
-      color: isEAForum ? theme.palette.grey[800] : undefined,
-      cursor: 'pointer'
-    }
+    color: isEAForum ? undefined : theme.palette.header.text,
```

Comment:
`IconButton` handles most of this styling for us. I set the color for LW to keep it looking the same as it does on their live site.